### PR TITLE
Fbe/flex support

### PR DESCRIPTION
**Pull request by fbeutin-ledger**

### CODE REVIEW

**Review by tdejoigny-ledger — COMMENTED** [review]
LGTM, just a small fix to do in ledger DB

File: src/handle_init_contract.c [hunk 1]
```diff
@@ -36,7 +27,7 @@ void handle_init_contract(void *parameters) {
     memset(context, 0, sizeof(*context));
 
     uint32_t selector = U4BE(msg->selector, 0);
-    if (find_selector(selector, CIAN_SELECTORS, NUM_SELECTORS, &context->selectorIndex)) {
+    if (!find_selector(selector, CIAN_SELECTORS, NUM_SELECTORS, &context->selectorIndex)) {
         msg->result = ETH_PLUGIN_RESULT_UNAVAILABLE;
         return;
```
Comment (tdejoigny-ledger) [inline src/handle_init_contract.c:32]: I don't understand how this plugin could have worked before? (before your fix)

Comment (apaillier-ledger) [inline src/handle_init_contract.c:32]: Return values in their own `find_selector` function :
* found : 0
* not found : -1

In the `find_selector` function from the plugin SDK:
* found : true
* not found : false

**Comment by apaillier-ledger** [comment]
> LGTM, just a small fix to do in ledger DB

Variant must be set to `CIAN`, to match the existing one.

**Review by tdejoigny-ledger — APPROVED** [review]

**Pull request merged**